requirements.txt

```diff
@@ -0,0 +1,4 @@
+mf2py==1.1.2
+requests==2.26.0
+beautifulsoup4==4.10.0
+lxml==4.7.1
```

Comment:
Tests failed without lxml installed throwing the following error and as such, I've added it to the requirements.

```
bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: lxml. Do you need to install a parser library?
```